_data/accessibility-tests/text-input.yml

```diff
@@ -0,0 +1,80 @@
+title: Text input
+component_status: pass
+test_items:
+# General tests
+  - summary: Instructions don't use visual cues.
+    summary_additional: |
+      Text input instructions don't rely on visual cues, such as "in the area below." 
+      If visual instructions exist, they are announced through an ARIA label.
```

Comment:
@amycole501 @alex-hull @sarah-sch 

This summary strikes me as contradictory. Reads a bit to me  like "Don't do this, BUT if you do make sure you do this other thing too". I might be misunderstanding! Is there another way to phrase this?

Reply:
I could see how this is confusing! Maybe we could change the wording to just include that second sentence. We ended up marking it as conditional, as it is dependent on the instructions the person is including. There is a conditional text as well as to why it would be marked as such.

Reply:
Looking at the [Understanding 1.3.3](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html) page, it doesn't mention including these instructions in the aria-label as a fix. It recommends including additional information so the instructions don't rely _solely_ on visual cues. Could we re-write this to something along the lines of:

> **Instructions don't rely solely on visual cues**
> If visual cues are used to describe controls, provide additional instructions other than shape and/or location to allow users to understand the instructions even if they cannot perceive shape and/or location.

Reply:
Maybe this: If you use visual cues like shapes or locations to describe controls, also provide extra instructions. This way, users who can't see shapes or locations can still understand how to use the controls.

Reply:
Hey @mahoneycm , here are my suggestions:

**Instructions don’t use visual cues.** Text input instructions are announced through an ARIA label. This way, users who can't see shapes or locations can still understand how to use the controls.

**Instructions use non-visual labels.** Text input instructions are announced through an ARIA label and can be understood by users who can’t see shapes or locations. Labels do not rely on visual cues like “in the area below”.

Reply:
Thanks for these @finekatie !

I think I'm leaning towards the second set of instructions with one minor change:

> **Instructions that use visual _~labels~_ cues also use non-visual information**
Text input instructions are announced through an ARIA label and can be understood by users who can’t see shapes or locations.

How does that sound @amycole501  @alex-hull  @amyleadem ?

Reply:
I like the change especially since not everyone will be clear about what constitutes a label.

Reply:
I agree, I think the description helps with understanding as well.

Reply:
Resolved in [6834c19](https://github.com/uswds/uswds-site/pull/2697/commits/6834c19b70ffa20d593c9f1ddfd654317645b07f)